static/js/modules/columns.js

```diff
@@ -506,6 +513,31 @@ var reports = {
   }
 };
 
+var debts = [
+  {
+    data: 'committee',
+    orderable: false,
+    className: 'all column--large',
+    render: function (data, type, row) {
+      return row.committee.name;
+    }
+  },
+  {
+    data: 'creditor_debtor_name',
+    orderable: false,
+    className: 'all column--large',
+  },
+  {
+    data: 'nature_of_debt',
+    orderable: false,
+    className: 'all column--large',
+  },
+  dateColumn({data: 'load_date', orderable: true, className: 'min-tablet hide-panel column--med'}),
```

Comment:
Paging @PaulClark2 : what's the logical value to show for a date column with debts? And what would that column be called in the database?